pkg/storaged/dialog.jsx

```diff
@@ -861,7 +861,7 @@ const TextInputCheckedComponent = ({ tag, val, title, update_function }) => {
             <Checkbox isChecked={val !== false}
                       id={tag}
                       label={title}
-                      onChange={checked => update_function(checked ? "" : false)} />
+                      onChange={(_event, checked) => update_function(checked ? "" : false)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)